visual-dotnet/SauceLabs.Visual/VisualClient.cs

```diff
@@ -47,6 +66,11 @@ public VisualClient(WebDriver wd, Region region, string username, string accessK
         /// <param name="buildOptions">the options of the build creation</param>
         public VisualClient(WebDriver wd, Region region, string username, string accessKey, CreateBuildOptions buildOptions)
```

Comment:
nice to have: 
we need to add a description for rest parameters here.

Reply:
Which rest parameters are you mentioning ?